packages/ecommerce-ui/components/FormComponents/TextFieldElement.tsx

```diff
@@ -1,4 +1,7 @@
 /* eslint-disable no-nested-ternary */
+import { useQuery } from '@graphcommerce/graphql'
+import { PasswordRequirements } from '@graphcommerce/magento-customer/components/SignUpForm/PasswordRequirements'
```

Comment:
This is giving an error, but @graphcommerce/magneto-customer may not be a dependency of ecommerce-ui.

The ecommerce-ui package is _not_ allowed to have backend specific code.
Create a `<MagentoPasswordElement/>` which uses this and applies the password requirements.